### ISSUE TITLE
Publish modules to sn0int.com

### ISSUE THREAD

**Issue by kpcyrd** [sysdogs/sn0int-modules#2]
heya!

found your repo and it seems you haven't published your module yet. Is there anything I can help with? :) I assume this is because you'd rather publish it on the sysdogs namespace (which isn't really supported/possible yet, considering it's an org)?

thanks!

**Comment by limakzi** [comment]
/x0! 
We have two other modules almost production ready (dnstrails subdomain search),  we would like to fix service mapping in `shodan-ports` and then publish them all. If you need some help with namespaces support, let us know!

Thank you for `sn0int`, we are always happy to contribute!

**Comment by kpcyrd** [comment]
Yes please! I've created https://github.com/kpcyrd/sn0int/issues/185 for discussion. :)
